openapi.yaml

```diff
@@ -872,59 +949,56 @@ definitions:
         type: number
         format: float
         description: The number of pays per calendar year.
+        example: 12.0
       employeeClassificationCode:
         type: string
+        description: The employee class attached to the job, based on the Employee Class entered on the Position Form (NBAPOSN).
+        example: CA
       employerCode:
         type: string
         description: Employer identification code.
+        example: ORST
       annualSalary:
         type: number
         format: float
-        example: 20050.35
-      earnCodeEffectiveDate:
-        type: string
-        format: date
-        description: The date the earnCode became effective for this job.
-        example: '2018-12-31'
+        description: Annual salary for the job.
+        example: 30000.50
       earnCode:
         type: string
         description: Default earnings code attached to the job and used on timesheet.
+        example: REG
       earnCodeHours:
         type: number
         format: float
         description: Hours/units defaulted on timesheets for earnCode.
+        example: 173.33
       earnCodeShift:
         type: string
         description: Shift code used as a default on timesheet for earnCode.
-      useTemporarySSN:
+      useTemporarySsn:
         type: boolean
         description: When creating or updating a job, this field indicates the employee needs a temporary social security number assigned.
-      employeeInformationRelease:
+      employeeInformationReleaseInd:
         type: boolean
         description: Indicates the employee has consented to having their employment information shared.
-      salaryInformationRelease:
+      salaryInformationReleaseInd:
         type: boolean
-        description: Indicates the employee has consented to having their salary information shared.
+        description: Indicates the employee accepted or denied to have their salary information shared.
       salaryInformationReleaseDate:
         type: string
         format: date
         description: The date that the employee accepted or denied to have their salary information shared.
```

Comment:
Can you add an example here? It's mainly for helping improve the UI of developer portal when we import this doc.